notifications/query.go

```diff
@@ -0,0 +1,80 @@
+package notifications
+
+import (
+	"encoding/json"
+
+	context "github.com/jbenet/go-ipfs/Godeps/_workspace/src/code.google.com/p/go.net/context"
+	peer "github.com/jbenet/go-ipfs/p2p/peer"
+)
+
+const RoutingQueryKey = "RoutingQueryEvent"
```

Comment:
a good pattern for that is:

```go
const routingQueryKey = "RoutingQueryEvent" // can these be types instead of strings?

// inside dht/ file:
func RegisterForNotifications(...) {
  notif.RegisterForNotifications(routingQueryKey)
}
```